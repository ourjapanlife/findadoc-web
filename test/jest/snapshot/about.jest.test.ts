import { mount } from '@vue/test-utils'
import { createI18n } from 'vue-i18n'
import About from '~/pages/about.vue'

const i18n = createI18n({
    legacy: false,
    locale: 'en-US',
    messages: { 'en-US': {} }
})

describe('about.vue', () => {
    it('renders correctly', () => {
        const wrapper = mount(About, {
            global: {
                plugins: [i18n]
            }
        })
        expect(wrapper.element).toMatchSnapshot()
    })
})
