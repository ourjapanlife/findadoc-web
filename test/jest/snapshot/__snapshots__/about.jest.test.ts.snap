// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`about.vue > renders correctly 1`] = `
<div
  class="overflow-y-auto w-full"
>
  <div
    class="flex flex-col items-center px-10 md:px-32"
  >
    <svg
      alt="pink heart with a white plus in the middle to symbolize health"
      class="my-4 h-32"
      fill="none"
      role="img"
      title="heart icon"
      viewBox="0 0 155 130"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="m81.561 127.8 60.214-60.214c14.794-14.868 16.949-39.176 2.974-54.713A38.65 38.65 0 0 0 117.022.015a38.655 38.655 0 0 0-28.398 11.297l-11.3 11.374-9.738-9.813C52.718-1.92 28.41-4.076 12.873 9.9a38.656 38.656 0 0 0-1.561 56.125L73.087 127.8a6.02 6.02 0 0 0 8.474 0Z"
        fill="url(#i-798740553__a)"
      />
      <path
        d="M72.293 68.238h-7.61a1.087 1.087 0 0 1-1.087-1.088v-6.522a1.087 1.087 0 0 1 1.087-1.087h7.61v-7.61a1.087 1.087 0 0 1 1.087-1.087h6.522a1.087 1.087 0 0 1 1.088 1.087v7.61h7.61a1.087 1.087 0 0 1 1.086 1.087v6.522a1.087 1.087 0 0 1-1.087 1.088h-7.61v7.61a1.087 1.087 0 0 1-1.087 1.087H73.38a1.087 1.087 0 0 1-1.087-1.088v-7.61Z"
        fill="#fff"
        stroke="#fff"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="2.174"
      />
      <circle
        cx="76.818"
        cy="64.066"
        r="23.333"
        stroke="#fff"
        stroke-width="1.556"
      />
      <circle
        cx="76.818"
        cy="64.066"
        r="23.333"
        stroke="#fff"
        stroke-width="1.556"
      />
      <defs>
        <lineargradient
          gradientUnits="userSpaceOnUse"
          id="i-798740553__a"
          x1="77.068"
          x2="77.068"
          y1="0"
          y2="129.543"
        >
          <stop
            stop-color="#EB8590"
          />
          <stop
            offset="1"
            stop-color="#CF616D"
          />
        </lineargradient>
      </defs>
    </svg>
    <h1
      class="mb-12 font-bold text-4xl"
      data-testid="about-heading"
    >
      about.heading
    </h1>
    <p
      class="mb-12 text-2xl"
      data-testid="about-subheading"
    >
      about.subheading
    </p>
    <p
      class="mb-6 text-md text-primary-text-muted"
      data-testid="about-paragraph1"
    >
      about.paragraph1
    </p>
    <p
      class="text-md text-primary-text-muted"
      data-testid="about-paragraph2"
    >
      about.paragraph2
    </p>
  </div>
  <div
    class="flex justify-center items-center"
    data-testid="members-header-container"
    id="members-header-container"
  >
    <div
      class="flex w-4/5 my-14"
      data-testid="members-header"
      id="members-header"
    >
      <div
        class="flex-1 border-currentColor/70 border self-center"
      />
      <div
        class="text-primary/80 text-xl font-semibold mx-4 p-2 whitespace-nowrap"
      >
        about.members
      </div>
      <div
        class="flex-1 border-currentColor/70 border self-center"
      />
    </div>
  </div>
  <div
    class="flex justify-center items-center"
    data-testid="members-container"
    id="members-container"
  >
    <div
      class="grid grid-cols-2 md:grid-cols-3 mx-4 gap-8 pb-12 md:pb-32"
      data-testid="members"
      id="members"
    >
      
      <div
        class="members-list grid"
        data-testid="member"
      >
        <div
          class="member flex flex-col items-center text-center"
        >
          
          <transition-stub
            appear="false"
            css="true"
            enteractiveclass="transition-opacity duration-1000 ease"
            enterfromclass="opacity-0"
            leaveactiveclass="transition-opacity duration-1000 ease"
            leavetoclass="opacity-0"
            persisted="true"
          >
            <div>
              <svg
                preserveAspectRatio="xMidYMid"
                viewBox="0 25 100 50"
                xmlns="http://www.w3.org/2000/svg"
              >
                <circle
                  cx="18"
                  cy="50"
                  fill="#e2985d"
                  r="4"
                >
                  <animate
                    attributeName="cy"
                    begin="0s"
                    calcMode="spline"
                    dur="1.4492753623188404s"
                    keySplines="0.5 0 0.5 1;0.5 0 0.5 1"
                    repeatCount="indefinite"
                    values="34;66;34"
                  />
                </circle>
                <circle
                  cx="27"
                  cy="61.314"
                  fill="#f3d974"
                  r="4"
                >
                  <animate
                    attributeName="cy"
                    begin="-0.18115942028985504s"
                    calcMode="spline"
                    dur="1.4492753623188404s"
                    keySplines="0.5 0 0.5 1;0.5 0 0.5 1"
                    repeatCount="indefinite"
                    values="34;66;34"
                  />
                </circle>
                <circle
                  cx="36"
                  cy="66"
                  fill="#f6e6e6"
                  r="4"
                >
                  <animate
                    attributeName="cy"
                    begin="-0.3623188405797101s"
                    calcMode="spline"
                    dur="1.4492753623188404s"
                    keySplines="0.5 0 0.5 1;0.5 0 0.5 1"
                    repeatCount="indefinite"
                    values="34;66;34"
                  />
                </circle>
                <circle
                  cx="45"
                  cy="61.314"
                  fill="#79ab81"
                  r="4"
                >
                  <animate
                    attributeName="cy"
                    begin="-0.5434782608695652s"
                    calcMode="spline"
                    dur="1.4492753623188404s"
                    keySplines="0.5 0 0.5 1;0.5 0 0.5 1"
                    repeatCount="indefinite"
                    values="34;66;34"
                  />
                </circle>
                <circle
                  cx="54"
                  cy="50"
                  fill="#e2985d"
                  r="4"
                >
                  <animate
                    attributeName="cy"
                    begin="-0.7246376811594202s"
                    calcMode="spline"
                    dur="1.4492753623188404s"
                    keySplines="0.5 0 0.5 1;0.5 0 0.5 1"
                    repeatCount="indefinite"
                    values="34;66;34"
                  />
                </circle>
                <circle
                  cx="63"
                  cy="38.686"
                  fill="#f3d974"
                  r="4"
                >
                  <animate
                    attributeName="cy"
                    begin="-0.9057971014492753s"
                    calcMode="spline"
                    dur="1.4492753623188404s"
                    keySplines="0.5 0 0.5 1;0.5 0 0.5 1"
                    repeatCount="indefinite"
                    values="34;66;34"
                  />
                </circle>
                <circle
                  cx="72"
                  cy="34"
                  fill="#f6e6e6"
                  r="4"
                >
                  <animate
                    attributeName="cy"
                    begin="-1.0869565217391304s"
                    calcMode="spline"
                    dur="1.4492753623188404s"
                    keySplines="0.5 0 0.5 1;0.5 0 0.5 1"
                    repeatCount="indefinite"
                    values="34;66;34"
                  />
                </circle>
                <circle
                  cx="81"
                  cy="38.686"
                  fill="#79ab81"
                  r="4"
                >
                  <animate
                    attributeName="cy"
                    begin="-1.2681159420289854s"
                    calcMode="spline"
                    dur="1.4492753623188404s"
                    keySplines="0.5 0 0.5 1;0.5 0 0.5 1"
                    repeatCount="indefinite"
                    values="34;66;34"
                  />
                </circle>
              </svg>
            </div>
          </transition-stub>
          <transition-stub
            appear="false"
            css="true"
            enteractiveclass="transition-opacity duration-1000 ease"
            enterfromclass="opacity-0"
            leaveactiveclass="transition-opacity duration-1000 ease"
            leavetoclass="opacity-0"
            persisted="true"
          >
            <img
              alt="LaShawn Toyoda"
              class="object-cover rounded-full w-32 h-32"
              data-testid="member-avatar-0"
              style="display: none;"
            />
          </transition-stub>
          
          <div
            class="member-name mt-1 text-xl font-bold"
            data-testid="member-name"
          >
            LaShawn Toyoda
          </div>
          <div
            class="member-title text-md text-primary-text-muted"
            data-testid="member-title"
          >
            Founder & Executive Director
          </div>
          <div
            class="member-social flex flex-row mt-2"
          >
            <a
              class="social-link"
              data-testid="member-linkedin"
              href="https://www.linkedin.com/in/lashawn-toyoda/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <svg
                alt="linkedin icon"
                class="social-icon w-8 h-8"
                role="img"
                title="linkedin icon"
                viewBox="0 0 72 72"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  fill="none"
                  fill-rule="evenodd"
                >
                  <path
                    d="M8 72h56a8 8 0 0 0 8-8V8a8 8 0 0 0-8-8H8a8 8 0 0 0-8 8v56a8 8 0 0 0 8 8Z"
                    fill="#007EBB"
                  />
                  <path
                    d="M62 62H51.316V43.802c0-4.99-1.896-7.777-5.845-7.777-4.296 0-6.54 2.901-6.54 7.777V62H28.632V27.333H38.93v4.67s3.096-5.729 10.453-5.729c7.353 0 12.617 4.49 12.617 13.777V62ZM16.35 22.794c-3.508 0-6.35-2.864-6.35-6.397C10 12.864 12.842 10 16.35 10c3.507 0 6.347 2.864 6.347 6.397 0 3.533-2.84 6.397-6.348 6.397ZM11.032 62h10.736V27.333H11.033V62Z"
                    fill="#FFF"
                  />
                </g>
              </svg>
            </a>
            <a
              class="social-link"
              data-testid="member-github"
              href="https://github.com/theyokohamalife/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <svg
                alt="github icon"
                class="social-icon w-8 h-8 ml-1"
                fill="none"
                role="img"
                title="github icon"
                viewBox="0 0 1024 1024"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M512 0C229.12 0 0 229.12 0 512c0 226.56 146.56 417.92 350.08 485.76 25.6 4.48 35.2-10.88 35.2-24.32 0-12.16-.64-52.48-.64-95.36-128.64 23.68-161.92-31.36-172.16-60.16-5.76-14.72-30.72-60.16-52.48-72.32-17.92-9.6-43.52-33.28-.64-33.92 40.32-.64 69.12 37.12 78.72 52.48 46.08 77.44 119.68 55.68 149.12 42.24 4.48-33.28 17.92-55.68 32.64-68.48-113.92-12.8-232.96-56.96-232.96-252.8 0-55.68 19.84-101.76 52.48-137.6-5.12-12.8-23.04-65.28 5.12-135.68 0 0 42.88-13.44 140.8 52.48 40.96-11.52 84.48-17.28 128-17.28 43.52 0 87.04 5.76 128 17.28 97.92-66.56 140.8-52.48 140.8-52.48 28.16 70.4 10.24 122.88 5.12 135.68 32.64 35.84 52.48 81.28 52.48 137.6 0 196.48-119.68 240-233.6 252.8 18.56 16 34.56 46.72 34.56 94.72 0 68.48-.64 123.52-.64 140.8 0 13.44 9.6 29.44 35.2 24.32C877.44 929.92 1024 737.92 1024 512 1024 229.12 794.88 0 512 0Z"
                  fill="#1B1F23"
                  fill-rule="evenodd"
                />
              </svg>
            </a>
          </div>
        </div>
      </div>
      <div
        class="members-list grid"
        data-testid="member"
      >
        <div
          class="member flex flex-col items-center text-center"
        >
          
          <transition-stub
            appear="false"
            css="true"
            enteractiveclass="transition-opacity duration-1000 ease"
            enterfromclass="opacity-0"
            leaveactiveclass="transition-opacity duration-1000 ease"
            leavetoclass="opacity-0"
            persisted="true"
          >
            <div>
              <svg
                preserveAspectRatio="xMidYMid"
                viewBox="0 25 100 50"
                xmlns="http://www.w3.org/2000/svg"
              >
                <circle
                  cx="18"
                  cy="50"
                  fill="#e2985d"
                  r="4"
                >
                  <animate
                    attributeName="cy"
                    begin="0s"
                    calcMode="spline"
                    dur="1.4492753623188404s"
                    keySplines="0.5 0 0.5 1;0.5 0 0.5 1"
                    repeatCount="indefinite"
                    values="34;66;34"
                  />
                </circle>
                <circle
                  cx="27"
                  cy="61.314"
                  fill="#f3d974"
                  r="4"
                >
                  <animate
                    attributeName="cy"
                    begin="-0.18115942028985504s"
                    calcMode="spline"
                    dur="1.4492753623188404s"
                    keySplines="0.5 0 0.5 1;0.5 0 0.5 1"
                    repeatCount="indefinite"
                    values="34;66;34"
                  />
                </circle>
                <circle
                  cx="36"
                  cy="66"
                  fill="#f6e6e6"
                  r="4"
                >
                  <animate
                    attributeName="cy"
                    begin="-0.3623188405797101s"
                    calcMode="spline"
                    dur="1.4492753623188404s"
                    keySplines="0.5 0 0.5 1;0.5 0 0.5 1"
                    repeatCount="indefinite"
                    values="34;66;34"
                  />
                </circle>
                <circle
                  cx="45"
                  cy="61.314"
                  fill="#79ab81"
                  r="4"
                >
                  <animate
                    attributeName="cy"
                    begin="-0.5434782608695652s"
                    calcMode="spline"
                    dur="1.4492753623188404s"
                    keySplines="0.5 0 0.5 1;0.5 0 0.5 1"
                    repeatCount="indefinite"
                    values="34;66;34"
                  />
                </circle>
                <circle
                  cx="54"
                  cy="50"
                  fill="#e2985d"
                  r="4"
                >
                  <animate
                    attributeName="cy"
                    begin="-0.7246376811594202s"
                    calcMode="spline"
                    dur="1.4492753623188404s"
                    keySplines="0.5 0 0.5 1;0.5 0 0.5 1"
                    repeatCount="indefinite"
                    values="34;66;34"
                  />
                </circle>
                <circle
                  cx="63"
                  cy="38.686"
                  fill="#f3d974"
                  r="4"
                >
                  <animate
                    attributeName="cy"
                    begin="-0.9057971014492753s"
                    calcMode="spline"
                    dur="1.4492753623188404s"
                    keySplines="0.5 0 0.5 1;0.5 0 0.5 1"
                    repeatCount="indefinite"
                    values="34;66;34"
                  />
                </circle>
                <circle
                  cx="72"
                  cy="34"
                  fill="#f6e6e6"
                  r="4"
                >
                  <animate
                    attributeName="cy"
                    begin="-1.0869565217391304s"
                    calcMode="spline"
                    dur="1.4492753623188404s"
                    keySplines="0.5 0 0.5 1;0.5 0 0.5 1"
                    repeatCount="indefinite"
                    values="34;66;34"
                  />
                </circle>
                <circle
                  cx="81"
                  cy="38.686"
                  fill="#79ab81"
                  r="4"
                >
                  <animate
                    attributeName="cy"
                    begin="-1.2681159420289854s"
                    calcMode="spline"
                    dur="1.4492753623188404s"
                    keySplines="0.5 0 0.5 1;0.5 0 0.5 1"
                    repeatCount="indefinite"
                    values="34;66;34"
                  />
                </circle>
              </svg>
            </div>
          </transition-stub>
          <transition-stub
            appear="false"
            css="true"
            enteractiveclass="transition-opacity duration-1000 ease"
            enterfromclass="opacity-0"
            leaveactiveclass="transition-opacity duration-1000 ease"
            leavetoclass="opacity-0"
            persisted="true"
          >
            <img
              alt="Philip Ermish"
              class="object-cover rounded-full w-32 h-32"
              data-testid="member-avatar-1"
              style="display: none;"
            />
          </transition-stub>
          
          <div
            class="member-name mt-1 text-xl font-bold"
            data-testid="member-name"
          >
            Philip Ermish
          </div>
          <div
            class="member-title text-md text-primary-text-muted"
            data-testid="member-title"
          >
            Tech Lead
          </div>
          <div
            class="member-social flex flex-row mt-2"
          >
            <a
              class="social-link"
              data-testid="member-linkedin"
              href="https://www.linkedin.com/in/philipermish"
              rel="noopener noreferrer"
              target="_blank"
            >
              <svg
                alt="linkedin icon"
                class="social-icon w-8 h-8"
                role="img"
                title="linkedin icon"
                viewBox="0 0 72 72"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  fill="none"
                  fill-rule="evenodd"
                >
                  <path
                    d="M8 72h56a8 8 0 0 0 8-8V8a8 8 0 0 0-8-8H8a8 8 0 0 0-8 8v56a8 8 0 0 0 8 8Z"
                    fill="#007EBB"
                  />
                  <path
                    d="M62 62H51.316V43.802c0-4.99-1.896-7.777-5.845-7.777-4.296 0-6.54 2.901-6.54 7.777V62H28.632V27.333H38.93v4.67s3.096-5.729 10.453-5.729c7.353 0 12.617 4.49 12.617 13.777V62ZM16.35 22.794c-3.508 0-6.35-2.864-6.35-6.397C10 12.864 12.842 10 16.35 10c3.507 0 6.347 2.864 6.347 6.397 0 3.533-2.84 6.397-6.348 6.397ZM11.032 62h10.736V27.333H11.033V62Z"
                    fill="#FFF"
                  />
                </g>
              </svg>
            </a>
            <a
              class="social-link"
              data-testid="member-github"
              href="https://github.com/ermish"
              rel="noopener noreferrer"
              target="_blank"
            >
              <svg
                alt="github icon"
                class="social-icon w-8 h-8 ml-1"
                fill="none"
                role="img"
                title="github icon"
                viewBox="0 0 1024 1024"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="M512 0C229.12 0 0 229.12 0 512c0 226.56 146.56 417.92 350.08 485.76 25.6 4.48 35.2-10.88 35.2-24.32 0-12.16-.64-52.48-.64-95.36-128.64 23.68-161.92-31.36-172.16-60.16-5.76-14.72-30.72-60.16-52.48-72.32-17.92-9.6-43.52-33.28-.64-33.92 40.32-.64 69.12 37.12 78.72 52.48 46.08 77.44 119.68 55.68 149.12 42.24 4.48-33.28 17.92-55.68 32.64-68.48-113.92-12.8-232.96-56.96-232.96-252.8 0-55.68 19.84-101.76 52.48-137.6-5.12-12.8-23.04-65.28 5.12-135.68 0 0 42.88-13.44 140.8 52.48 40.96-11.52 84.48-17.28 128-17.28 43.52 0 87.04 5.76 128 17.28 97.92-66.56 140.8-52.48 140.8-52.48 28.16 70.4 10.24 122.88 5.12 135.68 32.64 35.84 52.48 81.28 52.48 137.6 0 196.48-119.68 240-233.6 252.8 18.56 16 34.56 46.72 34.56 94.72 0 68.48-.64 123.52-.64 140.8 0 13.44 9.6 29.44 35.2 24.32C877.44 929.92 1024 737.92 1024 512 1024 229.12 794.88 0 512 0Z"
                  fill="#1B1F23"
                  fill-rule="evenodd"
                />
              </svg>
            </a>
          </div>
        </div>
      </div>
      <div
        class="members-list grid"
        data-testid="member"
      >
        <div
          class="member flex flex-col items-center text-center"
        >
          
          <transition-stub
            appear="false"
            css="true"
            enteractiveclass="transition-opacity duration-1000 ease"
            enterfromclass="opacity-0"
            leaveactiveclass="transition-opacity duration-1000 ease"
            leavetoclass="opacity-0"
            persisted="true"
          >
            <div>
              <svg
                preserveAspectRatio="xMidYMid"
                viewBox="0 25 100 50"
                xmlns="http://www.w3.org/2000/svg"
              >
                <circle
                  cx="18"
                  cy="50"
                  fill="#e2985d"
                  r="4"
                >
                  <animate
                    attributeName="cy"
                    begin="0s"
                    calcMode="spline"
                    dur="1.4492753623188404s"
                    keySplines="0.5 0 0.5 1;0.5 0 0.5 1"
                    repeatCount="indefinite"
                    values="34;66;34"
                  />
                </circle>
                <circle
                  cx="27"
                  cy="61.314"
                  fill="#f3d974"
                  r="4"
                >
                  <animate
                    attributeName="cy"
                    begin="-0.18115942028985504s"
                    calcMode="spline"
                    dur="1.4492753623188404s"
                    keySplines="0.5 0 0.5 1;0.5 0 0.5 1"
                    repeatCount="indefinite"
                    values="34;66;34"
                  />
                </circle>
                <circle
                  cx="36"
                  cy="66"
                  fill="#f6e6e6"
                  r="4"
                >
                  <animate
                    attributeName="cy"
                    begin="-0.3623188405797101s"
                    calcMode="spline"
                    dur="1.4492753623188404s"
                    keySplines="0.5 0 0.5 1;0.5 0 0.5 1"
                    repeatCount="indefinite"
                    values="34;66;34"
                  />
                </circle>
                <circle
                  cx="45"
                  cy="61.314"
                  fill="#79ab81"
                  r="4"
                >
                  <animate
                    attributeName="cy"
                    begin="-0.5434782608695652s"
                    calcMode="spline"
                    dur="1.4492753623188404s"
                    keySplines="0.5 0 0.5 1;0.5 0 0.5 1"
                    repeatCount="indefinite"
                    values="34;66;34"
                  />
                </circle>
                <circle
                  cx="54"
                  cy="50"
                  fill="#e2985d"
                  r="4"
                >
                  <animate
                    attributeName="cy"
                    begin="-0.7246376811594202s"
                    calcMode="spline"
                    dur="1.4492753623188404s"
                    keySplines="0.5 0 0.5 1;0.5 0 0.5 1"
                    repeatCount="indefinite"
                    values="34;66;34"
                  />
                </circle>
                <circle
                  cx="63"
                  cy="38.686"
                  fill="#f3d974"
                  r="4"
                >
                  <animate
                    attributeName="cy"
                    begin="-0.9057971014492753s"
                    calcMode="spline"
                    dur="1.4492753623188404s"
                    keySplines="0.5 0 0.5 1;0.5 0 0.5 1"
                    repeatCount="indefinite"
                    values="34;66;34"
                  />
                </circle>
                <circle
                  cx="72"
                  cy="34"
                  fill="#f6e6e6"
                  r="4"
                >
                  <animate
                    attributeName="cy"
                    begin="-1.0869565217391304s"
                    calcMode="spline"
                    dur="1.4492753623188404s"
                    keySplines="0.5 0 0.5 1;0.5 0 0.5 1"
                    repeatCount="indefinite"
                    values="34;66;34"
                  />
                </circle>
                <circle
                  cx="81"
                  cy="38.686"
                  fill="#79ab81"
                  r="4"
                >
                  <animate
                    attributeName="cy"
                    begin="-1.2681159420289854s"
                    calcMode="spline"
                    dur="1.4492753623188404s"
                    keySplines="0.5 0 0.5 1;0.5 0 0.5 1"
                    repeatCount="indefinite"
                    values="34;66;34"
                  />
                </circle>
              </svg>
            </div>
          </transition-stub>
          <transition-stub
            appear="false"
            css="true"
            enteractiveclass="transition-opacity duration-1000 ease"
            enterfromclass="opacity-0"
            leaveactiveclass="transition-opacity duration-1000 ease"
            leavetoclass="opacity-0"
            persisted="true"
          >
            <img
              alt="Russell Miller"
              class="object-cover rounded-full w-32 h-32"
              data-testid="member-avatar-2"
              style="display: none;"
            />
          </transition-stub>
          
          <div
            class="member-name mt-1 text-xl font-bold"
            data-testid="member-name"
          >
            Russell Miller
          </div>
          <div
            class="member-title text-md text-primary-text-muted"
            data-testid="member-title"
          >
            Healthcare Content Lead
          </div>
          <div
            class="member-social flex flex-row mt-2"
          >
            <a
              class="social-link"
              data-testid="member-linkedin"
              href="https://www.linkedin.com/in/russell-miller-51a5136b/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <svg
                alt="linkedin icon"
                class="social-icon w-8 h-8"
                role="img"
                title="linkedin icon"
                viewBox="0 0 72 72"
                xmlns="http://www.w3.org/2000/svg"
              >
                <g
                  fill="none"
                  fill-rule="evenodd"
                >
                  <path
                    d="M8 72h56a8 8 0 0 0 8-8V8a8 8 0 0 0-8-8H8a8 8 0 0 0-8 8v56a8 8 0 0 0 8 8Z"
                    fill="#007EBB"
                  />
                  <path
                    d="M62 62H51.316V43.802c0-4.99-1.896-7.777-5.845-7.777-4.296 0-6.54 2.901-6.54 7.777V62H28.632V27.333H38.93v4.67s3.096-5.729 10.453-5.729c7.353 0 12.617 4.49 12.617 13.777V62ZM16.35 22.794c-3.508 0-6.35-2.864-6.35-6.397C10 12.864 12.842 10 16.35 10c3.507 0 6.347 2.864 6.347 6.397 0 3.533-2.84 6.397-6.348 6.397ZM11.032 62h10.736V27.333H11.033V62Z"
                    fill="#FFF"
                  />
                </g>
              </svg>
            </a>
            <!--v-if-->
          </div>
        </div>
      </div>
      
    </div>
  </div>
</div>
`;

exports[`about.vue renders correctly 1`] = `
<div
  class="overflow-y-auto w-full"
>
  <div
    class="flex flex-col items-center px-10 md:px-32"
  >
    <!---->
    <h1
      class="mb-12 font-bold text-4xl"
      data-testid="about-heading"
    >
      about.heading
    </h1>
    <p
      class="mb-12 text-2xl"
      data-testid="about-subheading"
    >
      about.subheading
    </p>
    <p
      class="mb-6 text-md text-primary-text-muted"
      data-testid="about-paragraph1"
    >
      about.paragraph1
    </p>
    <p
      class="text-md text-primary-text-muted"
      data-testid="about-paragraph2"
    >
      about.paragraph2
    </p>
  </div>
  <div
    class="flex justify-center items-center"
    data-testid="members-header-container"
    id="members-header-container"
  >
    <div
      class="flex w-4/5 my-14"
      data-testid="members-header"
      id="members-header"
    >
      <div
        class="flex-1 border-currentColor/70 border self-center"
      />
      <div
        class="text-primary/80 text-xl font-semibold mx-4 p-2 whitespace-nowrap"
      >
        about.members
      </div>
      <div
        class="flex-1 border-currentColor/70 border self-center"
      />
    </div>
  </div>
  <div
    class="flex justify-center items-center"
    data-testid="members-container"
    id="members-container"
  >
    <div
      class="grid grid-cols-2 md:grid-cols-3 mx-4 gap-8 pb-12 md:pb-32"
      data-testid="members"
      id="members"
    >
      
      <div
        class="members-list grid"
        data-testid="member"
      >
        <div
          class="member flex flex-col items-center text-center"
        >
          <fadeintransitionimage
            alt="LaShawn Toyoda"
            border-radius="rounded-full"
            data-test-id="member-avatar-0"
            height="h-32"
            img-src="/avatars/lashawn_toyoda.jpg"
            object-styling="object-cover"
            width="w-32"
          />
          <div
            class="member-name mt-1 text-xl font-bold"
            data-testid="member-name"
          >
            LaShawn Toyoda
          </div>
          <div
            class="member-title text-md text-primary-text-muted"
            data-testid="member-title"
          >
            Founder & Executive Director
          </div>
          <div
            class="member-social flex flex-row mt-2"
          >
            <a
              class="social-link"
              data-testid="member-linkedin"
              href="https://www.linkedin.com/in/lashawn-toyoda/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <!---->
            </a>
            <a
              class="social-link"
              data-testid="member-github"
              href="https://github.com/theyokohamalife/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <!---->
            </a>
          </div>
        </div>
      </div>
      <div
        class="members-list grid"
        data-testid="member"
      >
        <div
          class="member flex flex-col items-center text-center"
        >
          <fadeintransitionimage
            alt="Philip Ermish"
            border-radius="rounded-full"
            data-test-id="member-avatar-1"
            height="h-32"
            img-src="/avatars/philip_ermish.jpg"
            object-styling="object-cover"
            width="w-32"
          />
          <div
            class="member-name mt-1 text-xl font-bold"
            data-testid="member-name"
          >
            Philip Ermish
          </div>
          <div
            class="member-title text-md text-primary-text-muted"
            data-testid="member-title"
          >
            Tech Lead
          </div>
          <div
            class="member-social flex flex-row mt-2"
          >
            <a
              class="social-link"
              data-testid="member-linkedin"
              href="https://www.linkedin.com/in/philipermish"
              rel="noopener noreferrer"
              target="_blank"
            >
              <!---->
            </a>
            <a
              class="social-link"
              data-testid="member-github"
              href="https://github.com/ermish"
              rel="noopener noreferrer"
              target="_blank"
            >
              <!---->
            </a>
          </div>
        </div>
      </div>
      <div
        class="members-list grid"
        data-testid="member"
      >
        <div
          class="member flex flex-col items-center text-center"
        >
          <fadeintransitionimage
            alt="Russell Miller"
            border-radius="rounded-full"
            data-test-id="member-avatar-2"
            height="h-32"
            img-src="/avatars/russell_miller.jpg"
            object-styling="object-cover"
            width="w-32"
          />
          <div
            class="member-name mt-1 text-xl font-bold"
            data-testid="member-name"
          >
            Russell Miller
          </div>
          <div
            class="member-title text-md text-primary-text-muted"
            data-testid="member-title"
          >
            Healthcare Content Lead
          </div>
          <div
            class="member-social flex flex-row mt-2"
          >
            <a
              class="social-link"
              data-testid="member-linkedin"
              href="https://www.linkedin.com/in/russell-miller-51a5136b/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <!---->
            </a>
            <!--v-if-->
          </div>
        </div>
      </div>
      
    </div>
  </div>
</div>
`;
